docs/source/en/api/pipelines/wuerstchen.md

```diff
@@ -0,0 +1,110 @@
+# Würstchen
+
+[Würstchen: Efficient Pretraining of Text-to-Image Models](https://huggingface.co/papers/2306.00637) is by Pablo Pernias, Dominic Rampas, and Marc Aubreville.
```

Comment:
Might be good to mention that that it's a discrete diffusion model. We don't have a lot of discrete diffusion models in `diffusers` :D

Reply:
It actually isn't a discrete diffusion model :c Paella was discrete, but also not a diffusion model in the classic DDPM sense. Würstchen works in continuous space, just like an LDM.

Reply:
Thanks for explaining.